source/extensions/filters/http/aws_lambda/config.h

```diff
@@ -27,6 +27,8 @@ namespace AwsLambda {
   COUNTER(fetch_failed)                                                        \
   COUNTER(fetch_success)                                                       \
   COUNTER(creds_rotated)                                                       \
+  COUNTER(webtoken_rotated)                                                    \
+  COUNTER(webtoken_failure)                                                    \
```

Comment:
Do you think this will be clear? I think a gauge here might be more useful. Is this thing ok or not. What do you think?

Reply:
Do you think it should be both? Long running state and current?

Reply:
Good question. I think that could be good. I like more info. So how about:
1. COUNTER: webtoken_rotated
2. COUNTER: empty_webtoken_failure
3. GAUGE: webtoken_state

Reply:
Updated